src/plugins/index_pattern_field_editor/public/lib/serialization.ts

```diff
@@ -19,7 +19,7 @@ export const deserializeField = (
 
   return {
     name: field.name,
-    type: field?.esTypes ? field.esTypes[0] : undefined,
+    type: field?.esTypes ? field.esTypes[0] : 'keyword',
```

Comment:
We need to find a better way to handle fields that have more than one type and fields that don't have a type. perhaps fields without a type should simply be excluded.

Reply:
I changed it because it did not think a field could exist without a ES type. Can you elaborate when that occurs?